src/lib.rs

```diff
@@ -1324,7 +1324,7 @@ impl<T> TrySendError<T> {
         }
     }
 
-    /// Returns `true` if the channel is closed.
+    /// Returns `true` if there are currently no active receivers, only inactive ones.
```

Comment:
Thanks for the contribution! However, I feel this is not really a correction but more like explanation. `InactiveReceiver` [docs](https://docs.rs/async-broadcast/latest/async_broadcast/struct.InactiveReceiver.html) mention that they're meant to keep the channel open and crate-level docs say:

> When all Senders or all Receivers are dropped, the channel becomes closed. When a channel is closed, no more messages can be sent, but remaining messages can still be received.

So I think there isn't any confusion possible.

Reply:
Thanks for the comment. Is there anything regarding the wording you would like me to improve to address your concerns? If you’re considering to not change the docs it may help to expand on my motivation for this change. 

I’m proposing the change because I was indeed confused by the documentation. I was looking for an alternative to matching for the `Inactive` variant of `TrySendError` and I noticed the `is_closed()` and `is_disconnected()` methods. Both of them have the same documentation but yield different results. For the `is_closed()` method it was clear to me that it returns `true` if the target holds the `Closed` variant. But for the `is_disconnected()` method I had to have a look at source code to confirm that it returns `true` if the target holds the `Inactive` variant.

Reply:
Oh I have to apologize. I had forgotten that we've a separate `is_closed` method. Your changes make sense then. Thanks.